onnxruntime/core/providers/cuda/cuda_execution_provider_info.cc

```diff
@@ -34,6 +34,7 @@ constexpr const char* kEnableSkipLayerNormStrictMode = "enable_skip_layer_norm_s
 constexpr const char* kPreferNHWCMode = "prefer_nhwc";
 constexpr const char* kUseEPLevelUnifiedStream = "use_ep_level_unified_stream";
 constexpr const char* kUseTF32 = "use_tf32";
+constexpr const char* kFuseBias = "fuse_bias";
```

Comment:
change the name to be more specified like `fuse_conv_bias`, it is because many operators can be fused with bias.